python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -388,6 +404,21 @@ void pyAltLaunchKernel(const std::string &name, MlirModule module,
   std::free(rawArgs);
 }
 
+/// @brief Serialize \p runtimeArgs into a flat buffer starting at
+/// \p startingArgIdx (0-based). This does not execute the kernel. This is
+/// useful for VQE applications when you want to serialize the constant
+/// parameters that are not being optimized. The caller is responsible for
+/// executing `std::free()` on the return value.
+void *pyGetKernelArgs(const std::string &name, MlirModule module,
+                      cudaq::OpaqueArguments &runtimeArgs,
+                      const std::vector<std::string> &names,
+                      std::size_t startingArgIdx) {
+  auto noneType = mlir::NoneType::get(unwrap(module).getContext());
+  auto [rawArgs, size, returnOffset] = pyAltLaunchKernelBase(
```

Comment:
Unlike Python, we cannot have multiple uses of `_` without getting compilation errors about duplicate variables. So in this case, I think it is probably better to simply leave their descriptive names (which are indeed accurate) rather than having something like `[rawArgs, _, __]`.